pubspec.yaml

```
@@ -4,8 +4,8 @@ version: 0.0.0
 author: Brandan Schmitz <brandan.schmitz@celestialdata.net>
 
 environment:
-  sdk: ">=2.8.3 <3.0.0"
+  sdk: "2.8.4"
```

Please leave this as a range versus a specific SDK version.
```suggestion
  sdk: ">=2.8.3<2.9.0"
```